framework/src/modules/interoperability/schemas.ts

```diff
@@ -93,6 +88,28 @@ export const ccmSchema = {
 	},
 };
 
+const chainValidators = {
```

Comment:
```suggestion
const chainValidatorsSchema = {
```

Reply:
renamed to `activeChainValidatorsSchema` since `chainValidatorsSchema` has been used elsewhere (lisk-sdk/framework/src/modules/interoperability/stores/chain_validators.ts@chainValidatorsSchema)